server/rpm/pbench-server.spec.j2

```diff
@@ -101,101 +101,22 @@ fi
 %posttrans
 
 %files
+
 %defattr(644, pbench, pbench, 755)
```

Comment:
What is the purpose/difference between the two mode-mask arguments to the `%defattr` directive?

Reply:
The second one is for directories; which is why there's no equivalent on `%attr`.